app/api/public/akatsuki.py

```diff
@@ -0,0 +1,31 @@
+"""\
+Provides an API exposing Akatsuki's beatmaps, which
+include internal state such as ranked status updates.
+"""
+
+import logging
+
+from fastapi import APIRouter
+from fastapi import Header
+from fastapi import Response
+
+from app.api.responses import JSONResponse
+from app.usecases import akatsuki_beatmaps
+
+router = APIRouter(tags=["Akatsuki Beatmaps"])
+
+
+@router.get(
+    "/api/akatsuki/v1/beatmaps/akatsuki_beatmaps_lookup",
```

Comment:
hmm we should use a path that represents the function being invoked, atm this says nothing about "custom ranked beatmaps"